lib/atlabs/SMS.js

```diff
@@ -0,0 +1,79 @@
+const request = require('./common/request');
+
+exports.SMSService = async data => {
+    try {
+        return await request.trigger({
+            method: 'POST',
+            path: '/atlabs/messaging',
+            credentials: { ...data },
+            payload: data.payload,
+        });
+    } catch (error) {
+        return error;
+    }
+};
+
+exports.PremiumSub = async data => {
```

Comment:
Avoid abbreviations, PremiumSub should be PremiumSubscription, apply the change to other method names below

Reply:
Edited everywhere